docbook/gnc-locale-C.dtd

```diff
@@ -39,171 +53,555 @@
 
 <!-- guimenu/guisubmenu/guimenuitem to be used as names in text -->
 <!-- <!ENTITY % gmi.xxx "<guimenuitem><accel></accel></guimenuitem>"> -->
-<!-- <!ENTITY gmi. "%gmi.;"> -->
+<!-- <!ENTITY gmi.xxx.yy-zz "%gmi.xxx.yy-zz;"> -->
+<!-- Description: 
+     gm:  short for <guimenu></guimenu>, every entry in the menubar.
+     gmi: short for <guimenuitem></guimenuitem>, each menu item, also in submenus
+     gsm: short for <guisubmenu></guisubmenu>, describes the menu entry of submenu
```

Comment:
How about `, the leaves of the menu tree`
```
    gm:  short for <guimenu></guimenu>, the entries in the menubar;
    gsm: short for <guisubmenu></guisubmenu>, the submenus;
    gmi: short for <guimenuitem></guimenuitem>, the leaves of the menu tree.

```
Perhaps one could add also `the labels of …`